synapse/cortex.py

```diff
@@ -1014,8 +1014,14 @@ async def initServiceStorage(self):
         await self._bumpCellVers('cortex:storage', (
             (1, self._storUpdateMacros),
             (2, self._storLayrFeedDefaults),
+            (3, self._viewNomergeToProtected),
         ), nexs=False)
 
+    async def _viewNomergeToProtected(self):
+        for view in self.listViews():
+            nomerge = view.info.get('nomerge')
```

Comment:
should we pop the old value here? currently the old value will still present in the definition but not get updated so it could potentially be out of sync if they grab the definition with `view.pack()` which seems more confusing than it not being present